docs/src/tutorials/getting_started.ipynb

```diff
@@ -75,24 +68,17 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "bb47d14c",
-   "metadata": {
-    "execution": {
-     "iopub.execute_input": "2023-09-18T16:02:36.408502Z",
-     "iopub.status.busy": "2023-09-18T16:02:36.408269Z",
-     "iopub.status.idle": "2023-09-18T16:02:36.427955Z",
-     "shell.execute_reply": "2023-09-18T16:02:36.427545Z"
-    }
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Generate a synthetic dataset\n",
     "timestamps = np.arange(0, 100, 0.1)\n",
     "n = len(timestamps)\n",
     "noise = 0.1 * np.random.randn(n)\n",
     "trend = 0.01 * timestamps\n",
-    "season = 0.4 * np.sin(0.5 * timestamps)\n",
+    "season = 0.4 * np.sin(timestamps)\n",
     "\n",
-    "raw_data = pd.DataFrame(\n",
+    "df_signals = pd.DataFrame(\n",
```

Comment:
Nit: It took me sometime to figure that "df" is dataframe :)
Maybe adding a note would help.